db/default/users.rb

```diff
@@ -1,17 +1,11 @@
-require 'highline/import'
-
 # see last line where we create an admin if there is none, asking for email and password
 def prompt_for_admin_password
   if ENV['ADMIN_PASSWORD']
     password = ENV['ADMIN_PASSWORD'].dup
-    say "Admin Password #{password}"
+    puts "Admin Password #{password}"
   else
-    password = ask('Password [test123]: ') do |q|
-      q.echo = false
-      q.validate = /^(|.{5,40})$/
-      q.responses[:not_valid] = 'Invalid password. Must be at least 5 characters long.'
-      q.whitespace = :strip
-    end
+    print "Password [test123]: "
+    password = STDIN.gets.chomp
```

Comment:
Ideally we would not `strip` or `chomp` either because as you said spaces at the end are also valid. But as this seeds are only meant for local development we can consider this an non-issue for now.